packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -390,11 +391,13 @@ const AutocompleteInput: FunctionComponent<
                                 htmlFor: id,
                             })}
                             helperText={
-                                <InputHelperText
-                                    touched={touched}
-                                    error={error}
-                                    helperText={helperText}
-                                />
+                                disableHelperText ? null : (
```

Comment:
Why do you introduce another prop for that? Wouldn't it be better to accept `false` for `helperText` ?

Reply:
See https://github.com/marmelab/react-admin/pull/4390

Reply:
Didn't see this one (I was thinking all filters was always broken due to ReferenceInput issue)
Will change my PR